linodecli/overrides.py

```diff
@@ -37,3 +41,65 @@ def handle_domains_zone_file(operation, output_handler, json_data) -> bool:
     """
     print("\n".join(json_data["zone_file"]))
     return False
+
+
+@output_override("linodes", "types", OutputMode.table)
+def handle_types_region_prices_list(
+    operation, output_handler, json_data
+) -> bool:
+    # pylint: disable=unused-argument
+    """
+    Override the output of 'linode-cli linodes types' to display regional pricing.
+    """
+    if len(json_data["data"]) < 1:
+        return True
+
+    output = Table()
+    headers = sorted(json_data["data"][0].keys() - ["addons"], key=len)
```

Comment:
To make things a bit more readable, we might want to add some logic here to ensure `region_prices` is the last column